Index: qrencode/configure.ac
===================================================================
--- qrencode.orig/configure.ac
+++ qrencode/configure.ac
@@ -156,6 +156,7 @@ CFLAGS="-Wall $CFLAGS"
 
 AC_OUTPUT
 
+if test x$verbose = xyes; then
 echo ""
 echo "Options used to compile and link:"
 echo "  CC       = $CC"
@@ -165,3 +166,4 @@ echo "  CXX      = $CXX"
 echo "  CXXFLAGS = $CXXFLAGS"
 echo "  LDFLAGS  = $LDFLAGS"
 echo ""
+fi
